app/main.py

```diff
@@ -2,0 +1,8 @@
+def copy_file(command: str) -> None:
+    _, target, new_name = command.split()
+
+    if target == new_name:
+        return
```

Comment:
The check `if target == new_name` prevents copying a file onto itself, which is good. However, it might be beneficial to provide a warning or log message to inform the user why the operation was skipped.